packages/pixeloven/cli-addon-stylelint/src/types.ts

```diff
@@ -0,0 +1,13 @@
+import { PixelOvenToolbox } from "@pixeloven/cli";
+
+interface RunResponse {
+    stdout?: Buffer | string;
+    status: number;
+    error?: Error;
```

Comment:
lol, alpha sorted the others above but not this one ;)

Reply:
Only what the linter tells me ;)